src/pvHelpers/cs_client/v4/mail.py

```diff
@@ -69,11 +69,11 @@ def dupeMessages(self, user, src_mailbox_id, dest_mailbox_id, uids):
 
         return data
 
-    @params(object, {LocalUser, UserDBNode}, {int, long})
-    def getMailHistory(self, user, last_rev_id):
+    @params(object, {LocalUser, UserDBNode}, {int, long}, unicode, unicode, unicode)
+    def getMailHistory(self, user, last_rev_id, for_user_id, for_user_cid, export_id):
         url, raw_body, headers = self.prepareSignedRequest(
-            user,  u"/mail/{}/mailboxes".format(user.mail_cid),
-            "GET", None
+            user,  u"/mail/{}/mailboxes".format(for_user_cid),
+            "GET", None, False, (export_id, for_user_id)
```

Comment:
additional params should be optional so to change the behavior of the call for export.